pygame2_extra_deps.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "pygame2_extra_deps",
+  "buildsystem": "simple",
+  "build-commands": [],
+  "modules": [
+    {
+      "name": "libmodplug_sdl_vendored",
+      "buildsystem": "cmake",
+      "config-opts": [ "-DBUILD_SHARED_LIBS=ON", "-DFOO=BAR" ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/libsdl-org/libmodplug",
+          "commit": "f3173c24dcce28d54429099c555665a59d729546"
+        }
+      ]
+    },
+    {
+      "name": "FluidSynth",
+      "buildsystem": "cmake",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/FluidSynth/fluidsynth",
+          "commit": "bfe3599b286f30d561e4df0b0785fa78d6652628",
+          "tag": "v2.3.0",
+          "x-checker-data": {
+            "type": "git",
+            "tag-pattern": "^v([\\d.]+)$"
+          }
+        }
+      ]
+    },
+    {
+      "name": "SDL2_mixer",
```

Comment:
SDL2_mixer is in the runtime

Reply:
It's built without support for MOD and MIDI. I tried removing the custom SDL2_mixer after everything was working and lost the music.